3rdparty/Makefile

```diff
@@ -1,3 +1,14 @@
+#
+# Note that this Makefile is borked and it will rebuild everything every time you run it. This is due
+# me depending on $(BUILD_DIR) everywhere.
```

Comment:
There should be some better way to solve it but I'm out of my makefile budget for this week :)